hygiene.md

```diff
@@ -58,3 +58,18 @@ Ausserdem:
    - Einmal-Hygiene-Tücher, um viel benutzte Gegenstände zu reinigen
    - Reinigungsmittel
    - Einmal-Handtücher zum Nachfüllen
+ - Für den offenen Samstag sind folgende zusätzliche Regelungen getroffen
+   - Am Eingang befindet sich ein Hinweis, auf die Aufsicht zu warten, die einen festen Werkbereich zuweist
+   - Trennlinien sorgen dafür das in der Warteschlange an der Tür der Abstand eingehalten wird
+   - Die Aufsicht hat Zugang zum Werkstatt Telefon (App) um Anfragen zur Belegung zu beantworten
+   - Auf der Webseite befindet sich ein Hinweis, das die Gäste am selben Tag Infos zur Belegung per Telefon bekommen können
```

Comment:
Halte ich für das Hygienekonzept nicht für relevant. Könnte man als Ergänzung zum Werkstattbetrieb festhalten.

Reply:
Ist ja auch unter dem seperaten Abschnitt. Ist schon relevant in dem Sinne, das wir Warteschlangen vermeiden.